easybuild/framework/easyblock.py

```diff
@@ -1445,7 +1445,20 @@ def extensions_step(self, fetch=False):
 
     def package_step(self):
         """Package software (e.g. into an RPM)."""
-        pass
+        rpmname = "HPCBIOS.20150211-%s-%s" % (self.name, self.version)
```

Comment:
variable name should not be `rpmname` but rather `pkgname` (so it's neutral).
the `HPCBIOS.20150211` should come from a variable (`pkgprefix`). but you might better use a template with named templates (it will make more sense this way, and then we read this in from a configuration option.

so something like
```python
pkgtemplate = "HPCBIOS.20150211-%(name)s-%(version)s"

pkgname=pkgtemplate % {
    'name:' self.name,
    'version' : self.version,
}
```